submissions/ustymchyk/memory-pair-game/index.css

```diff
@@ -0,0 +1,102 @@
+*, *::before, *::after {
+  box-sizing: border-box;
+}
+
+:root {
+  --grey: #263238;
+  --yellow: rgb(247, 202, 75);
+  --blue: rgb(64, 98, 163);
+}
+
+body {
```

Comment:
Now your game field has quite big height on my screen. I'd recommend you decreasing it and adjust to the screen height.
<img width="1432" alt="Screenshot 2020-12-02 at 12 29 01" src="https://user-images.githubusercontent.com/13544983/100862093-6391b200-349b-11eb-97ed-2b81870502a6.png">